app/src/main/res/layout/activity_explicit_intent.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".explicitIntent.ExplicitActivity">
+
+    <TextView
+        android:id="@+id/Text_title_explicit"
```

Comment:
Name not in Android convetion should be text_title_explicit not a **T**ext_title_explicit